W1-M6-object-oriented/LineOfCreditAccount.cs

```diff
@@ -0,0 +1,51 @@
+namespace classes
+{
+    /// <summary> 
+    /// Line of credit account class
+    /// </summary> 
+    public class LineOfCreditAccount : BankAccount
+    {
+
+        /// <summary> 
+        /// Method for line of credit account
```

Comment:
If you think about how this is shown, a better comment would be 

Creates a new LineOfCreditAccount